spec/controllers/wishlists_controller_spec.rb

```diff
@@ -63,7 +61,7 @@
 
       it "doesn't create new wishlist" do
         expect do
-          post :create, params: { wishlist: invalid_parameters }
+          post :create, params: { wishlist: attributes_for(:invalid_wishlist) }
         end.to_not change(Wishlist, :count)
```

Comment:
While you're at it. This fits in one line (we've got 120 chars limit):

    expect { post ... }.not_to change(Wishlist, :count)